test/team_statistics_test.rb

```diff
@@ -0,0 +1,144 @@
+require './test/test_helper'
+require './lib/stat_tracker'
+
+class TeamStatisticsTest < Minitest::Test
```

Comment:
I like the amount of tests and agree with the note about maybe adding more assertions.